### PR TITLE
ci: goreleaser and update formule brew

### PR DESCRIPTION
**Pull request by maxwelbm**
…action push on main

# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Auth
- [x] Infra
- [ ] Ledger
- [x] Mdz
- [ ] Transaction
- [ ] Audit
- [x] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [ ] I have updated the documentation accordingly.
- [ ] I have added necessary comments to the code, especially in complex areas.
- [ ] I have ensured that my changes adhere to the project's coding standards.
- [ ] I have checked for any potential security issues.
- [ ] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [ ] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please, always remember to target your PR to develop branch instead of main.

### CODE REVIEW

**Comment by maxwelbm** [comment]
@chenrui333

File: .goreleaser.yml [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
-      - "6"
+      - "7"
+
     ldflags:
-      - -s -w # Flags to reduce binary size
+      - -s -w -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientID=9670e0ca55a29a466d31
+      - -s -w -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientSecret=dd03f916cacf4a98c6a413d9c38ba102dce436a9
+      - -s -w -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.URLAPIAuth=http://127.0.0.1:8080
+      - -s -w -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.URLAPILedger=http://127.0.0.1:3000
+      - -s -w -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.Version={{.Version}}
```
Comment (chenrui333) [inline .goreleaser.yml]:
```suggestion
      - -s -w 
      - -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientID=9670e0ca55a29a466d31
      - -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientSecret=dd03f916cacf4a98c6a413d9c38ba102dce436a9
      - -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.URLAPIAuth=http://127.0.0.1:8080
      - -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.URLAPILedger=http://127.0.0.1:3000
      - -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.Version={{.Version}}
```

File: .goreleaser.yml [hunk 2]
```diff
@@ -1,19 +1,34 @@
-project_name: mdz
+version: 2
 
 builds:
   - id: "mdz"
+    env:
+      - CGO_ENABLED=0
     main: ./components/mdz/main.go
```
Comment (chenrui333) [inline .goreleaser.yml]:
```suggestion
    main: ./components/mdz
```

**Review by chenrui333 — APPROVED** [review]

**Review by MartinezAvellan — APPROVED** [review]

**Review by lfbarrile01 — APPROVED** [review]
Approved.

**Pull request merged**